nntrainer/graph/graph_node.h

```diff
@@ -39,7 +39,8 @@ class GraphNode {
    * This ensures that the operations are executed in the order of their
    * listing.
    */
-  typedef std::tuple<unsigned int, unsigned int, unsigned int, unsigned int> ExecutionOrder;
+  typedef std::tuple<unsigned int, unsigned int, unsigned int, unsigned int>
```

Comment:
Wouldn't it be more concise if changed unsigned int to uint?

Reply:
we are using unsigned int so many point in NNTrainer. may be separate PR might better to change everywhere.